src/common/prop.c

```diff
@@ -342,13 +342,19 @@ daos_prop_valid(daos_prop_t *prop, bool pool, bool input)
 			break;
 		case DAOS_PROP_CO_CSUM_CHUNK_SIZE:
 			/** Accepting anything right now */
+			val = prop->dpp_entries[i].dpe_val;
+			if (val >= (1ULL << 32)) {
```

Comment:
I would just remove comment above as error message provides enough info. But isn't 1<<32 4GiB?

Reply:
Yes, it is encoded on 32 bits in the pool service. Any concerns? We can impose a lower value if you prefer.

Reply:
No, I think that's fine. The error message below says "Should be < 2GiB" which confused me.